src/App.js

```diff
@@ -1,29 +1,20 @@
-import logo from './logo.svg';
 import './App.css';
+import React from 'react';
+import Calculator from './Components/calculator';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <div className="container">
+        <Calculator />
+      </div>
+    );
+  }
```

Comment:
Great job @ritikarawat220 🤝

- [ ] [Optional] I would recommend you make this component a functional component as well.

Reply:
Thanks @1GORDON  for approving.. I have made the required changes..